### PR TITLE
added a footer with navbar

### PR DESCRIPTION
**Pull request by Robert-Schurig**

### CODE REVIEW

File: styles/GlobalStyles.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import {createGlobalStyle} from "styled-components";
 
 const GlobalStyles = createGlobalStyle`
-    html,
+    /* html, */
```
Comment (stefankorder) [inline styles/GlobalStyles.js]: Please delete unused code.

File: styles/GlobalStyles.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
     body {
         padding: 0;
         margin: 0;
         font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+        /* display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: row; */
+  /* height: 80%;
+  width: 70%;    */
```
Comment (stefankorder) [inline styles/GlobalStyles.js]: Please delete unused code.

**Review by stefankorder — APPROVED** [review]
LGTM! Please take a look at my comments.

File: styles/GlobalStyles.js (hunk 2 above)
Comment (Robert-Schurig) [reply styles/GlobalStyles.js]: Done. Thanks for the hint.

**Pull request merged**